academic_helper/logic/shnaton_parser.py

```diff
@@ -324,11 +314,24 @@ def create_course_groups(
     group, created = ClassGroup.objects.get_or_create(
         occurrence=occurrence, class_type=group_class_type, mark=group_mark
     )
+    if teachers and created:
+        group.teachers.add(*teachers)
+    if not created and teachers and set(teachers) != set(group.teachers.all()):
+        group.teachers.set(teachers)
+        log.info(f"Group {group.id} was updated")
```

Comment:
I think after these you need to call `group.save()`, right?

Reply:
I don't think, it was created before.